### PR TITLE
Api authorization

### PR DESCRIPTION
**Pull request by ZaradarBH**
- In the current version we will apply Authorize filter at the controller level for Topic + Capability controllers.

- Fixed problem with integration/unit test not using MockAuth scheme in test project.

### CODE REVIEW

**Comment by ZaradarBH** [comment]
Dont merge this without my approval :)

File: src/CapabilityService.WebApi/Startup.cs
Comment (kim-lindhard-dfds) [inline src/CapabilityService.WebApi/Startup.cs:94]: Is the intention to remove this functionality?

File: src/CapabilityService.Tests/Builders/HttpClientBuilder.cs
Comment (kim-lindhard-dfds) [inline src/CapabilityService.Tests/Builders/HttpClientBuilder.cs:44]: Didnt you already do this by letting the StubStartUp inherit from the webapi startup class?

**Review by kim-lindhard-dfds — APPROVED** [review]

**Review by SEQUOIIA — APPROVED** [review]
Looks reasonable to me, although I've been staring at this code so much the last couple of days that I'm probably having a hard time spotting any obvious flaws.

Should wait to merge this though until https://github.com/dfds/blaster/pull/134 and ADSync has the ability to handle auth.

**Pull request merged**
